### PR TITLE
Quickstart to run docsy-example locally using a Docker container

### PR DESCRIPTION
**Pull request by geriom**
Added a quickstart to get new users up and running in just a few minutes. This PR is contingent on https://github.com/google/docsy-example/pull/75 being merged.

### CODE REVIEW

File: userguide/content/en/docs/Quickstart/_index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+provides a basic site that you can use as starting point to create your own
+documentation.
+
+1. Download a copy to your computer:
+
+  ```bash
+  git clone --recurse-submodules --depth 1 https://github.com/google/docsy-example.git
+  ```
+
+1. Change your working directory to the newly crated folder:
```
Comment (LisaFC) [inline userguide/content/en/docs/Quickstart/_index.md:31]: crated -> created

File: userguide/content/en/docs/Quickstart/_index.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+
+---
+title: "Quickstart"
```
Comment (LisaFC) [inline userguide/content/en/docs/Quickstart/_index.md:3]: I think as it is, this is slightly confusing - we have "Quickstart" and "Getting started" as sort of peer pages and it's not clear which one you should go to first (I also have a personal dislike of the term Quickstart but I can live with it. :) ). We also should make it clear that you can use the Docker image as a real environment for working locally on your own sites, not just as a "kick the tyres with the example site" tool for a tutorial.

How about we move this under /getting_started/ and rename it "Quickstart with Docker" (so it'll show up as a sub-page in the nav)

I would also in the main Getting started add a new section under Prerequisites and installation (right at the top so users who don't want to have to install anything can see it) - something like

### Use our Docker image

We provide a [Docker](linky) image that you can use to run and test your Docsy site locally, without having to install all Docsy's dependencies.

You can see how to get started with this approach by following our Quickstart tutorial (linky). If you don't want to use Docker, follow the instructions below to install Hugo and PostCSS.

Comment (geriom) [reply userguide/content/en/docs/Quickstart/_index.md:3]: I like your suggestions. I just reviewed the [docker-compose PR](https://github.com/google/docsy-example/pull/93) and suggested a couple of changes, as soon as we get that merged I will update this PR.

Comment (LisaFC) [reply userguide/content/en/docs/Quickstart/_index.md:3]: Would be great to get this in now that the Docker update is merged! *subtle hint face*

Comment (geriom) [reply userguide/content/en/docs/Quickstart/_index.md:3]: Currently working on it :)

**Comment by geriom** [comment]
Updated installation instructions with the requested changes.

**Comment by LisaFC** [comment]
This is great! I fixed a few nits directly, so it's good to go!

**Pull request merged**